robottelo/vm.py

```diff
@@ -716,37 +716,27 @@ def configure_rhai_client(self, activation_key, org, rhel_distro):
             system to satellite
         :param org: The org to which the system is required to be registered
         :param rhel_distro: rhel distribution for
+        :param register: Whether to register client to insights
         :return: None
         """
         # Download and Install ketello-ca rpm
         self.install_katello_ca()
         self.register_contenthost(org, activation_key)
 
-        # Red Hat Access Insights requires RHEL 6/7 repo and it is not
+        # Red Hat Access Insights requires RHEL 6/7/8 repo and it is not
         # possible to sync the repo during the tests, Adding repo file.
         if rhel_distro == DISTRO_RHEL6:
             rhel_repo = settings.rhel6_repo
-            insights_repo = settings.rhai.insights_client_el6repo
         if rhel_distro == DISTRO_RHEL7:
             rhel_repo = settings.rhel7_repo
-            insights_repo = settings.rhai.insights_client_el7repo
+        if rhel_distro == DISTRO_RHEL8:
+            rhel_repo = settings.rhel8_repo
```

Comment:
just a suggestion, Can we put all the supported distro in the dictionary, I think it will help to reduce the expansion of "if statement'?
```
supported_distro_map = {DISTRO_RHEL6: settings.rhel6_repo, DISTRO_RHEL7: settings.rhel7_repo,
                                DISTRO_RHEL8: settings.rhel8_repo}
        if rhel_distro in supported_distro_map:
            rhel_repo = supported_distro_map[rhel_distro]
```

Reply:
Done.

What keeps these ifs from ever expanding is Red Hat support policy for RHEL - with new RHEL every 3-4 years and each RHEL being supported for up to 14 years (in ELS), there are only 4 major RHEL releases that are ever supported at any given time.